lib/beaker/options/command_line_parser.rb

```diff
@@ -98,6 +98,12 @@ def initialize
             @cmd_options[:timeout] = value
           end
 
+          opts.on '-r RUNNER', '--runner RUNNER',
+                  'Specify which test runner to use',
+                  'supported runners: beaker, minitest' do |value|
```

Comment:
The minitest runner itself doesn't look like it runs at this point, is that true?  I'm fine with that being the case, but I'd like to update this line that says it can until it's true.